packages/atomic/src/components/insight/insight-icon-button/insight-icon-button.tsx

```diff
@@ -0,0 +1,26 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {Button, ButtonProps} from '../../common/button';
+interface Props extends Partial<ButtonProps> {
+  icon: string;
+  ariaLabel: string;
+  tooltip?: string;
+  onClick?: () => void;
+}
+
+export const InsightIconButton: FunctionalComponent<Props> = ({
+  icon,
+  ariaLabel,
+  tooltip,
+  onClick = () => {},
+}) => (
+  <Button
+    style="outline-neutral"
+    ariaLabel={ariaLabel}
+    class="p-3"
+    part="button"
+    onClick={onClick}
+  >
+    <atomic-icon icon={icon} class="w-4 h-4 shrink-0"></atomic-icon>
```

Comment:
since the button has an aria label, I would recommend adding `aria-hidden="true"` to the icon.